packages/atomic/src/components/common/atomic-icon/atomic-icon.tsx

```diff
@@ -122,12 +122,10 @@
 
   public render() {
     if (this.error) {
-      // deepcode ignore FormatString: client-side code.
       console.error(this.error, this.host);
       this.host.remove();
       return;
     }
-    // deepcode ignore ReactSetInnerHtml: This is not React code, deepcode ignore DOMXSS: Value escaped in upstream code
-    return <Host innerHTML={this.svg}></Host>;
+    return <Host innerHTML={this.svg} aria-hidden="true"></Host>;
```

Comment:
@olamothe so here, instead of using the deepcode comment, just dismiss the alert from the PR, and you'll never hear about it ever again ;)
Process will be identical when we move to CodeQL